optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -897,6 +862,45 @@ def _create_objective(
             pbar=pbar,
         )
 
+    def get_best_booster(self) -> "lgb.Booster":
+        """Return the best booster.
+
+        If the best booster cannot be found, :class:`ValueError` will be raised. To prevent the
+        errors, please save boosters by specifying the ``model_dir`` argument of
+        :meth:`~optuna.integration.lightgbm.LightGBMTuner.__init__`,
+        when you resume tuning or you run tuning in parallel.
+        """
+        if self._best_booster_with_trial_number is not None:
+            if self._best_booster_with_trial_number[1] == self.study.best_trial.number:
+                assert isinstance(self._best_booster_with_trial_number[0], lgb.Booster)
```

Comment:
This assertion might not be necessary anymore.
```suggestion
```

Reply:
You're right. I remove the sentence.